src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -220,7 +220,7 @@ namespace AirflowNetwork {
     // Link simulation variable in air distribution system
     // Sensible and latent exchange variable in air distribution system
 
-    void generic_crack(Real64 &coef,             // Flow coefficient
+    void generic_crack(Real64 const &coef,       // Flow coefficient
```

Comment:
There is no sense in passing scalars by `const &`, just pass them by value. See [here](https://github.com/NREL/EnergyPlus/wiki/Parameters-and-Return-Values).

Reply:
OK, agreed, so changes should be a little less robotic and a little more thoughtful.

    [src/EnergyPlus/AirflowNetwork/src/Elements.cpp:3503]:(style),[constParameter],Parameter 'coefficient' can be declared as reference to const

     3503   void generic_crack(Real64 &coefficient,      // Flow coefficient